### PR TITLE
Fix unused variable warnings in libgdiplus

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

**Comment by hughbe** [comment]
@akoeplinger any ideas? I don't see how this PR can build locally but fail in CI

**Comment by filipnavara** [comment]
@hughbe Did you merge/rebase on top of master? I'd assume it's the usual case of building the merged tree of master + PR, not the PR branch alone...

**Comment by hughbe** [comment]
I did, yes

File: src/Makefile.am [hunk 1]
```diff
@@ -129,6 +129,6 @@ endif
 
 libgdiplus_la_LIBADD = $(GDIPLUS_LIBS)
 
-AM_CPPFLAGS = $(GDIPLUS_CFLAGS) -Wall -Wno-unused -Wno-format
+AM_CPPFLAGS = $(GDIPLUS_CFLAGS)
```
Comment (filipnavara) [inline src/Makefile.am]: This seems to trigger the build errors, btw.

Comment (filipnavara) [reply src/Makefile.am]: The 'gdip_setpixel_32bppARGB' function is really unused. Somehow `-Wunused` is apparently already included in the warnings and this removed `-Wno-unused` then triggers the build error.

**Comment by hughbe** [comment]
@akoeplinger @filipnavara great i've got it building successfully! would like to merge so I can fix the conflicts in https://github.com/mono/libgdiplus/pull/511

**Pull request merged**